index.js

```diff
@@ -36,25 +30,19 @@ http
           case '/auth':
             switch (req.method) {
               case 'POST':
-                result = auth.handler(req, res);
+                result = auth.handler(req);
                 break;
               case 'GET':
-                result = auth.status(params);
-                break;
-              case 'DELETE':
-                result = auth.logout(req, res);
+                result = auth.status(req);
                 break;
             }
             break;
         }
+        body = Buffer.from(JSON.stringify(result));
 
-        res.writeHead(result.status, {'Content-Type': 'text/html'});
+        res.writeHead(result.status, result.headers)
```

Comment:
i could make an argument for writing the `content-type` header in here if it's always going to be json instead of over in `auth`.